### PR TITLE
FEAT: Added Navigation component

### PR DESCRIPTION
**Pull request by kotesh-arya**
Screenshot from 2022-02-16 23-51-46

### CODE REVIEW

**Review by Rockyraven — APPROVED** [review]
All goods to me go ahead

File: Components/Navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+                    sandbox="allow-scripts allow-same-origin">
+                  </iframe>
+                </div>
+            </article>
+           
+            <div class="section-footer">
+                <a href="/Components/Grid/grid.html">
+                    < Grid</a>
```
Comment (yashpatel1797) [inline Components/Navigation/navigation.html:91]: check here.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to me. added one suggestion.

File: Components/Navigation/navigation.html (hunk 1 above)
Comment (kotesh-arya) [reply Components/Navigation/navigation.html:91]: That's actually left arrow option for the previous page

**Pull request merged**